frontend/src/utils/entity.ts

```diff
@@ -10,6 +10,7 @@ export const videoFromRelatedEntity = (entity: RelatedEntityObject): Video => {
     views: entity.metadata.views,
     uploader: entity.metadata.uploader,
     language: entity.metadata.language,
+    tournesol_score: null,
```

Comment:
the `tournesol_score` can be used in the `VideoSerializerWithCriteria` intead of `VideoSerializer` to avoid this addition

Reply:
Is there an obstacle that prevents this change?

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/614/commits/0b4be828aaa764bebbccf09b7474ce6e220fb559.